### ISSUE TITLE
Norm的实现

### ISSUE THREAD

**Issue by wangyirui** [intro-llm/intro-llm.github.io#66]
在2.1.4部分公示2.11前面用LN表示直观理解是LayerNorm，transformer里也的确用的layernorm。但是下面的代码实现里norm类的参数size给的是d_model而不是shape，所以看起来实现的是batch norm（对batch的channel进行归一化）而不是对每个sample进行归一化。能否解答一下这里是否有typo？谢谢！

`class Norm(nn.Module):
      def __init__(self, d_model, eps = 1e-6):
          super().__init__()
          **self.size = d_model**
          # 层归一化包含两个可以学习的参数
          self.alpha = nn.Parameter(torch.ones(self.size))
          self.bias = nn.Parameter(torch.zeros(self.size))
          self.eps = eps
     def forward(self, x):
          norm = self.alpha * (x - x.mean(dim=-1, keepdim=True))
          / (x.std(dim=-1, keepdim=True) + self.eps) + self.bias
          return norm`
